append_lifecycle.go

```diff
@@ -207,6 +226,31 @@ func NewAppender(ctx context.Context, d Driver, opts *AppendOptions) (*Appender,
 	return a, t.Shutdown, r, nil
 }
 
+func followerStats(ctx context.Context, f Follower, size func(context.Context) (uint64, error)) {
+	name := f.Name()
+	t := time.NewTicker(200 * time.Millisecond)
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		case <-t.C:
+		}
+
+		idx, err := f.EntriesProcessed(ctx)
```

Comment:
```suggestion
		count, err := f.EntriesProcessed(ctx)
```

If I was really pedantic...

Reply:
if I _were_ really pedantic... 🤣